applications/zigbee_weather_station/README.rst

```diff
@@ -135,6 +135,7 @@ Power switch (SW1):
 Button (SW3):
     Used to start and cancel the Identify mode.
     Additionally, pressing the button indicates user input, which allows a Sleepy End Device to start the network rejoin procedure if it was stopped (for example, after timeout).
+    If pressed for one second, it initiates the factory reset of the device. Releasing the button within the one-second window cancels the factory reset procedure.
```

Comment:
```suggestion
Button (SW3):
    Depending on how long the button is pressed:
    
    * If pressed for less than one second, it starts and cancels the Identify mode.
    Additionally, pressing the button indicates user input, which allows a Sleepy End Device to start the network rejoin procedure if it was stopped (for example, after timeout).
    * If pressed for one second, it initiates the factory reset of the device. 
       Releasing the button within the one-second window cancels the factory reset procedure.
```
Few questions:
1) Do I understand correctly that the Identify mode is triggered if the button is pressed for less than 1 second, also including the "one-second window" you mention in the last sentence?
2) Can the factory reset be started at any moment? This is especially important here, as one button is used for different purposes.
3) Is the factory reset window a sharp one second or "one second or more"?

Reply:
@greg-fer
Ad.1. Identification start/stop is triggered as soon as button is pressed (pushed). Button release event is not taken into account here.
Ad. 2. Factory reset can be started at anymoment.
Ad. 3. When button is pushed, a timer is started. When 1s elapses, there is a check if the button is still pressed. If yes, FR is done.

Reply:
Based on our chat on Teams, I suggest:
```suggestion
Button (SW3):
    Depending on how long the button is pressed:
    
    * If pressed for less than one second and released, it starts or cancels the Identify mode.
    * If pressed for one second or more, it initiates the factory reset of the device. 
    
    Additionally, pressing the button shortly or releasing it before one second has elapsed indicates user input, which allows a Sleepy End Device to start the network rejoin procedure if it was previously stopped (for example, after timeout).
```